webapp/helpers.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from hashlib import md5
+from datetime import datetime
+
+
+def get_current_year():
+    """Returns current year"""
+    now = datetime.now()
+    return now.year
+
+
+def versioned_static(filename):
```

Comment:
This function should be shared with [snapcraft.io's version](https://github.com/canonical-web-and-design/snapcraft.io/blob/master/webapp/template_utils.py#L58), so I guess it should be in a module somewhere. But also, @hatched was looking into a new generic solution to this, so that should inform this module. For now I've just created [an issue on snapcraft.io](https://github.com/canonical-web-and-design/snapcraft.io/issues/1940).

Reply:
I was hoping this would be included in our future flask base/core module, with other common helpers. Same goes for the template finder view and things we plan to re-use a lot.